boards/espressif/esp32c6_devkitc/esp32c6_devkitc_lpcore_defconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2025 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+# Single-threaded, no timer support in the kernel
+CONFIG_MULTITHREADING=n
+CONFIG_NUM_PREEMPT_PRIORITIES=0
+CONFIG_SYS_CLOCK_EXISTS=n
+
+# Dynamic memory
+CONFIG_KERNEL_MEM_POOL=n
+CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
+
+# Interrupts
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+
+# Memory protection
+CONFIG_THREAD_STACK_INFO=n
+CONFIG_THREAD_CUSTOM_DATA=n
+
+# Boot
+CONFIG_BOOT_BANNER=n
+
+# Console
+CONFIG_PRINTK=n
+
+# Build
+CONFIG_SIZE_OPTIMIZATIONS=y
+
+CONFIG_GEN_ISR_TABLES=n
```

Comment:
some of these seem to be soc specific so should be set in Kconfig.defconfig at soc level. For malloc size, does having that enabled cause the build to fail to run/why is it being set to 0? Likewise with boot banner, printk, etc. why and these being forced off?

Reply:
In LP Core we have only 16KB of memory. We set these configurations off to reduce the app size. We set `CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE` to 0 because we want to avoid `malloc_prepare` at the moment. Heap mgmt will come in later PRs. Printk, banner, etc are not set because there are no UART support, these and other peripherals support will come in later PRs also.